micro_brewery/data/res_config_settings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_config_account_setting_micro_brewery" model="res.config.settings">
+        <field name="group_analytic_accounting" eval="1"/>
+        <field name="group_stock_production_lot" eval="1"/>
+        <field name="group_mrp_wo_shop_floor" eval="1"/>
+        <field name="group_mrp_routings" eval="1"/>
+        <field name="group_product_variant" eval="1"/>
+        <field name="group_uom" eval="1"/>
+        <field name="group_discount_per_so_line" eval="1"/>
```

Comment:
```suggestion
```

Reply:
according to main db it is been enabled so cannot be removed